### ISSUE TITLE
Lint failure for Helm 3

### ISSUE THREAD

**Issue by nbhargava** [PostHog/charts#4]
The PostHog chart currently fails the lint check with helm v3.2.1.

```
$ helm lint posthog
==> Linting posthog
[INFO] Chart.yaml: icon is recommended
[ERROR] Chart.yaml: chart type is not valid in apiVersion 'v1'. It is valid in apiVersion 'v2'
[ERROR] Chart.yaml: dependencies are not valid in the Chart file with apiVersion 'v1'. They are valid in apiVersion 'v2'
```

In theory, things with `[ERROR]` are supposed to represent issues that would manifest as installation failures, but I don't really know that this is the case here.

Easiest fix is to bump `apiVersion` from `v1` to `v2`, but this wouldn't work if you were opting to maintain Helm 2/3 compatibility.

**Comment by mariusandra** [comment]
@fuziontech can you assist here? I built the charts for Helm v3 (apiVersion v2), as it seemed like a wise version to target. Then I noticed GitLab's [PostHog intergation](https://gitlab.com/gitlab-org/cluster-integration/cluster-applications/-/merge_requests/74) required a Helm 2 (apiVersion v1) chart.

Just downgrading the `apiVersion` to v1 and keeping `Chart.*`, `requirements.lock` + the `charts` folder seemed to satisfy everyone: I could install the chart locally with helm 3 and it worked in GitLab's helm 2 system.

Darned linters :D.

Should we have separate v1 vs v2 apiVersion charts? What's the best way to clean this up?

**Comment by fuziontech** [comment]
For now I would focus support V1 api since Helm V3 fully supports V1 api. As soon as Gitlab upgrades to supporting V2 (Helm V3) I would jump to that version and deprecate the old one. We don't loose much focusing on V1 right now though.

This might be a way to kind of support both for now:
https://github.com/PostHog/charts/pull/5

**Comment by mariusandra** [comment]
Thank you, merged! 
To release a new chart we use [this procedure](https://posthog.com/handbook/engineering/release-new-version) (end of page). Is this a good way to manage charts? There might be some cli tooling that I'm not aware of for example...
I guess this chart update requires a version bump. What's the convention here `1.2.0` or `2.0.0`?

Feel free to merge PRs as well!

**Comment by fuziontech** [comment]
Yes! This procedure seems reasonable to me.
For the chart update I would bump the minor version and keep the major version to 1, but that is totally by taste. Anything works. From what I've seen the versioning is left up to the owner of the chart.

**Comment by mariusandra** [comment]
Thanks @fuziontech ! I also [updated the guidelines](https://github.com/PostHog/posthog.com/pull/92) to follow when releasing a new posthog version and added a note about editing `ChartV3.yaml` (cc @timgl ).

@nbhargava we released version `1.2.0` of the chart, which should pass the linter! Please let us know if that turns out not to be the case.

**Comment by nbhargava** [comment]
Yes, this works great. Thanks all!
